src/routes/transactions/mappers/common/native-staking.mapper.spec.ts

```diff
@@ -533,9 +625,18 @@ describe('NativeStakingMapper', () => {
         .with('product_type', 'dedicated')
         .build();
       const networkStats = networkStatsBuilder().build();
-      const validatorPublicKey = faker.string.hexadecimal({
-        length: KilnDecoder.KilnPublicKeyLength * 2,
-      }); // 2 validators
+      const validators = [
+        faker.string.hexadecimal({
+          length: KilnDecoder.KilnPublicKeyLength,
+          // Transaction Service returns _publicKeys lowercase
+          casing: 'lower',
+        }),
+        faker.string.hexadecimal({
+          length: KilnDecoder.KilnPublicKeyLength,
+          casing: 'lower',
+        }),
+      ] as Array<`0x${string}`>;
+      const validatorPublicKey = concat(validators);
```

Comment:
Nit: `validators` is not being expected in the response, I think it would be better to include it:

```typescript
      expect(actual).toEqual(
        expect.objectContaining({
          type: 'NativeStakingWithdraw',
          value: withdrawalEventParams.rewards.toString(),
          tokenInfo: {
            address: NULL_ADDRESS,
            decimals: chain.nativeCurrency.decimals,
            logoUri: chain.nativeCurrency.logoUri,
            name: chain.nativeCurrency.name,
            symbol: chain.nativeCurrency.symbol,
            trusted: true,
          },
          validators,
        }),
      );
```

Reply:
Added in 9dd2b7a708e9ba95104617eb3113bdfea4a30455.